experiments/link20_imp_onpolicy.py

```diff
@@ -13,12 +13,12 @@
 import policies
 from task import LinearLQRValuePredictionTask
 
-dim=30
+dim = 20
 gamma = 0.95
 sigma = np.ones(2*dim)*1.
 dt = 0.1
 mdp = examples.NLinkPendulumMDP(np.ones(dim)*.5, np.ones(dim)*.6, sigma=sigma, dt=dt)
-phi = features.squared_tri(dim*(dim+1)/2+1)
+phi = features.squared_diag(2*dim)
```

Comment:
This is the same as in the corresponding off-policy version (https://github.com/chrodan/tdlearn/blob/7ad9c0e2485a86897a11e8c3c2bba2460efe3cd7/experiments/link20_imp_offpolicy.py#L20)